lib/galaxy/tool_util/cwl/util.py

```diff
@@ -328,9 +332,10 @@ def _ensure_file_exists(file_path):
 
 class FileLiteralTarget:
 
-    def __init__(self, contents, **kwargs):
+    def __init__(self, contents, path=None, **kwargs):
         self.contents = contents
         self.properties = kwargs
+        self.path = path
 
```

Comment:
unrelated to this PR, but the __str__ method of this class requires path to have a value